lms/djangoapps/bulk_email/api.py

```diff
@@ -48,3 +51,39 @@ def get_unsubscribed_link(username, course_id):
     optout_url = reverse('bulk_email_opt_out', kwargs={'token': token, 'course_id': course_id})
     url = f'{lms_root_url}{optout_url}'
     return url
+
+
+def create_course_email(course_id, sender, targets, subject, html_message, text_message=None, template_name=None,
```

Comment:
Python API function made to align ourselves closer to [OEP-49](https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0049-django-app-patterns.html#data-py). We can use this to cleanup the import and direct use of the `CourseEmail` model in external apps.